examples/restaurantbot/restaurant_domain.yml

```diff
@@ -51,7 +51,7 @@ templates:
   utter_ask_location:
     - "where?"
   utter_ask_moreupdates: 
-    - "if you'd like to modify anything else, please tell me what"
+    - "OK making a reservation for a restaurant in {location} (price: {price}, cuisine: {cuisine}) for {people} people."
```

Comment:
why did you change "if you'd like to modify anything else, please tell me what" with "OK making a reservation ... "?